### ISSUE TITLE
Play Button in "Sora Stream" Provider

### ISSUE THREAD

**Issue by intellimediatv** [stantanasi/streamflix#107]
### Description

Sometimes on initial click the content does not play gives an error message then u have to click it 2x or 3x for the movie or tv series to actually play 

### Reproduction steps

Go to Movie or TV Series select "Watch"
Initial click doesnt play have to click additional times for the player to play this occurs from time to time.
Does not apply to every movie or tv series.

### App version

Version 1.6.2 (tv) version

### Operating system

NVIDIA SHIELD TV ANDROID OS 11

### Please confirm the following

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] If related to a provider, I have checked the site and it works, but not the app.

**Comment by oathzed** [comment]
Yeah this does happen a lot to me as well. It's odd cause it works after you press the button a few times like you said

**Comment by stantanasi** [comment]
Closing this issue because it seems to be resolved even though I didn't do anything 🤷🏽
